dids/src/main/kotlin/web5/sdk/dids/methods/DidUtil.kt

```diff
@@ -0,0 +1,37 @@
+package web5.sdk.dids.methods
+
+
+/**
+ * Utl methods for DIDs.
+ *
+ */
+public object DidUtil {
```

Comment:
I generally a little allergic to anything named "utility" because it's a bit amorphous in my mind

What is a `SimpleDid`? It may warrant it's own place in the code base next to `PortableDid` and `BearerDid`? (doesn't need to be addressed here, food for thought)

Reply:
(which I know `PortableDid` and `BearerDid` aren't here yet, so looking ahead)

Reply:
actually i don't have to use this class or its methods anymore

i can use DidUri instead of SimpleDid!

Reply:
i also dislike `utility` anything but alas my brainjuice had ran out by the time i wrote this